mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -707,6 +707,10 @@ gvnic {
     that represents whether the underlying node VMs are spot. See the [official documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/spot-vms)
     for more information. Defaults to false.
 
+* `tags` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) A list of tags to
```

Comment:
Isn't this already documented below?

Reply:
Forgot about that. Reverting last commit

Reply:
Reverted